tools/testsuite/__init__.py

```diff
@@ -46,7 +46,7 @@
         },
         "arch": {
             "help": "The onedir artifact architecture",
-            "choices": ("x86_64", "aarch64", "amd64", "x86"),
+            "choices": ("x86_64", "arm64", "amd64", "x86"),
```

Comment:
"choices": ("x86_64", "aarch64", "amd64", "arm64", "x86")

Reply:
We don't care about `aarch64` in this context.